net/config.go

```diff
@@ -24,6 +24,7 @@ type Options struct {
 	EnableRelay       bool
 	GRPCServerOptions []grpc.ServerOption
 	GRPCDialOptions   []grpc.DialOption
+	BootstrapPeers    []string
```

Comment:
question: Sorry if I've misread, but I can't spot where this is actually read - is it dead code?

Reply:
It's used here: https://github.com/sourcenetwork/defradb/blob/f8032b3d912597833f747665774c927e38343408/net/peer.go#L84

Reply:
cheers!